experimental/shortvec/Cargo.toml

```diff
@@ -0,0 +1,43 @@
+# This file is part of ICU4X. For terms of use, please see the file
+# called LICENSE at the top level of the ICU4X source tree
+# (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+[package]
+name = "shortvec"
```

Comment:
issue: this is `shortboxedslice`

Reply:
I was thinking about naming the crate `shortvec` but the type inside `ShortBoxedSlice`, potentially adding `ShortSlice` in the future

Reply:
`shortvec` still wouldn't be the right name if you have two slice types in there. There is not `ShortVec` type in the future, because there's already `SmallVec`.